MM2RandoLib/Resources/SpritePatches/WeaponSpriteRandomizer.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using MM2Randomizer.Patcher;
+using MM2Randomizer.Random;
+
+namespace MM2Randomizer.Resources.SpritePatches
+{
+    public static class WeaponSpriteRandomizer
+    {
+        public static void ApplySprites(ISeed in_Seed, Patch in_Patch, String in_TargetFileName)
```

Comment:
It seems like these `FooSpriteRandomizer` files have the same interface? Could we refactor `ApplySprites` for example, to be in a base class. Then each of these classes would just list all the arrays and override some instance variable of the base class for the list of lists (`FOO_SPRITE_COLLECTION`).